automation_tools/__init__.py

```diff
@@ -1186,11 +1185,22 @@ def ak_install(admin_password=None, clean_beaker=True,
         '--activationkey="{1}"'.format(org, act_key)
     )
     run(cmd)
+
+    # Refresh subscriptions and clean up YUM
     print('Refreshing Subscription-manager.')
     run('subscription-manager refresh')
     print('Performing yum clean up.')
     run('yum clean all', quiet=True)
 
+    # Set yum stdout log level to be less verbose
+    set_yum_debug_level()
+
+    # Install some basic packages
+    update_basic_packages()
```

Comment:
Doing this will re-subscribe the box to CDN as it also calls in subscribe and unsubscribe within it.
You may want to drop this.